dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -440,11 +443,131 @@ class ApplicationsControllerTest < ::ActionController::TestCase
             district_name: 'A School District',
             school_name: 'Hogwarts',
             email: 'minerva@hogwarts.edu',
-          }.stringify_keys, JSON.parse(@response.body).first
+          }.stringify_keys, JSON.parse(@response.body)['applications'].first
         )
       end
     end
 
+    test 'cohort view returns regional partner cohort capacity for teacher applications' do
+      time = Date.new(2017, 3, 15)
+
+      Timecop.freeze(time) do
+        application = create(
+          :pd_teacher1819_application,
+          course: 'csp',
+          regional_partner: @regional_partner,
+          user: @serializing_teacher,
+        )
+
+        application.update_form_data_hash({first_name: 'Minerva', last_name: 'McGonagall'})
+        application.save
+        application.update(status: 'accepted')
```

Comment:
Use bang versions in case of error. Also since we're saving anyway, maybe reorder to prevent an extra sql call:

```ruby
application.status = 'accepted'
application.save!
```